src/Illuminate/Foundation/Auth/AuthenticatesUsers.php

```diff
@@ -99,6 +99,10 @@ protected function sendLoginResponse(Request $request)
         $request->session()->regenerate();
 
         $this->clearLoginAttempts($request);
+        
+        if ($request->expectsJson()) {
+            return response()->json([$this->username() => trans('auth.success')], 200);
```

Comment:
Well this file and files closely related to authentication both have (non 200) status codes, so i personally prefer if ether all have codes, or none do.